File: Sunset/Basics/CODE_CHALLENGES/Day 2/exercise5.js
/*
Write a program that adds the digits in a two-digit number
*/

function digitAdd(num) {
  // Convert the number to a string to access individual digits
  const numStr = num.toString();

  // Extract the first and second digits and convert them back to numbers
  const firstDigit = parseInt(numStr[0]);
  const secondDigit = parseInt(numStr[1]);

  // Add the two digits together
  const sum = firstDigit + secondDigit;

  return sum;
}

File: Maame Esi/Exercise5.py
#Write a program that adds the digits in a two-digit number.


digit = input('two digit number: ')

first = digit[0]
second = digit[1]


if len(digit) == 2: 
    if first.isnumeric() and second.isnumeric():          
        sum = int(first) + int(second)
        print(sum)
    else: 
        print('the input is not numeric')
else: 
    print('digits is longer than two')


